addons/website_event_booth/static/src/scss/website_event_booth.scss

```diff
@@ -71,3 +71,13 @@ label.o_wbooth_category_unavailable {
 .o_wevent_booth_category {
     background-color: $o-wevent-bg-color-light;
 }
+
+.o_wevent_cta  {
```

Comment:
If it's always the same, can't we put it at the root to avoid copying it everywhere? website_event will always be installed for these modules so maybe just have it there?